policyengine_us/variables/gov/ed/pell_grant/efc/pell_grant_simplified.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_us.model_api import *
+
+
+class pell_grant_simplified(Variable):
+    value_type = bool
+    entity = Person
+    label = "Use Pell Grant simplified formula"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        tax_unit = person.tax_unit
+        head_income = tax_unit("pell_grant_head_income", period)
+        max_income = parameters(period).gov.ed.pell_grant.efc.simplified.max_income
+        income_eligible = head_income < max_income
+        medicaid = add(tax_unit, period, ["medicaid"])
```

Comment:
Make a list parameter for the qualifying benefits. You should then be able to pass them all to `add` at once.

Reply:
Some benefits are different entity types. Does that matter?

Reply:
No, adds and add sum across entity types